deploy/ndm-operator.yaml

```diff
@@ -164,77 +667,104 @@ spec:
         hostPath:
           path: /var/openebs/sparse
 ---
+# Create NDM operator deployment
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: openebs-ndm-operator
+  name: node-disk-operator
   namespace: openebs
-  labels:
-    name: openebs-ndm-operator
-    openebs.io/component-name: ndm-operator
-    openebs.io/version: 2.6.0
 spec:
+  replicas: 1
+  strategy:
+    type: Recreate
   selector:
     matchLabels:
-      name: openebs-ndm-operator
-      openebs.io/component-name: ndm-operator
+      name: node-disk-operator
+  template:
+    metadata:
+      labels:
+        name: node-disk-operator
+    spec:
+      serviceAccountName: openebs-maya-operator
+      containers:
+      - name: node-disk-operator
+        image: openebs/node-disk-operator:ci
+        ports:
+        - containerPort: 8080
+          name: liveness
+        imagePullPolicy: IfNotPresent
+        readinessProbe:
+          exec:
+            command:
+            - stat
+            - /tmp/operator-sdk-ready
+          initialDelaySeconds: 4
+          periodSeconds: 10
+          failureThreshold: 1
+        env:
+        - name: WATCH_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        # the service account of this pod
+        - name: SERVICE_ACCOUNT
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.serviceAccountName
+        - name: OPERATOR_NAME
+          value: "node-disk-operator"
+        - name: CLEANUP_JOB_IMAGE
+          value: "openebs/linux-utils:ci"
+        - name: OPENEBS_IO_INSTALL_CRD
+          value: "false"
+       # OPENEBS_IO_IMAGE_PULL_SECRETS environment variable is used to pass the image pull secrets
+       # to the cleanup pod launched by NDM operator
+       #- name: OPENEBS_IO_IMAGE_PULL_SECRETS
+       #  value: ""
+---
+# Create NDM cluster exporter deployment.
+# This is an optional component and is not required for the basic
+# functioning of NDM
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ndm-cluster-exporter
+  namespace: openebs
+spec:
```

Comment:
Is the exporter required?

Reply:
no we can remove that, i did remove the exporter daemonset but missed this.